### ISSUE TITLE
Question about the inference  after finetuning the TTM with covariate and id_columns

### ISSUE THREAD

**Issue by ZhangAllen98** [ibm-granite/granite-tsfm#403]
Suppose that we have a dataset, where column is like ['time',  'A', 'B', 'C', 'D', 'id', 'target'], which means we want to predict the 'target' of different site 'id' using the other covariates.   

I want to know if we can using a part of sites to fine-tune  the TTM model, and used the fine-tuned model to predict the other site's target just like zero-shot forecast. If so, how to set 'id_columns' in fine-tune process? Could you give some example? Thanks.

In the fine-tune process with covariates,  we need use the `TimeSeriesPreprocessor` to  normalize the data. I want to know when using the fine-tuned model to do forecast task, should we load the `tsp` to scaling  the input data. If so, what is the right way to load the `TimeSeriesPreprocessor`?
